styles/_app.scss

```diff
@@ -128,6 +128,13 @@
 // hide filter categories on mobile
 @media only screen and (max-width: 750px) {
   .categories-filter {
+    width: 100%;
+    padding: 0 !important;
```

Comment:
Using `!important` generously can be a slippery slope. Can we achieve the same result without that?

Reply:
I want to make it look like the current version. Keep the style that all of you have designed. Can compare at https://electron-website-pr-952.herokuapp.com/apps and https://electronjs.org/apps (browser width is small)

Reply:
Let see, what I do just to make should be nothing different from the current design